chia/wallet/wallet_user_store.py

```diff
@@ -55,7 +55,7 @@ async def _clear_database(self):
 
     async def create_wallet(
         self, name: str, wallet_type: int, data: str, id: Optional[int] = None, in_transaction=False
-    ) -> Optional[WalletInfo]:
+    ) -> WalletInfo:
```

Comment:
Sweet! I was going to attempt this after getting https://github.com/Chia-Network/chia-blockchain/pull/10893 in.

Feel free to go ahead of me. I can resolve the conflicts in my branch.